frontend/src/metabase/services.js

```diff
@@ -481,7 +481,7 @@ export const SetupApi = {
 
 export const UserApi = {
   create: POST("/api/user"),
-  list: GET("/api/user"),
+  list: GET("/api/user/recipients"),
```

Comment:
Let's definitely get a FE review on this change. Not sure what the implications are.

Reply:
Didn't see anyone use UserApi.list(), I used for testing purposes but can change it back if needed